scripts/variantstore/beta_docs/gvs-quickstart.md

```diff
@@ -92,29 +93,13 @@ If you’ve done this correctly, you should see your Terra proxy group listed in
 
 ### Step 7. Clone the GVS beta workspace
 
-The GVS beta workspace in Terra is read-only, so you’ll need to clone the workspace to create a copy where you can upload your own data and run the workflow. Clone the workspace using the billing project you created in Step 2 (above) by following the instructions in the article [Make your own project workspace](https://support.terra.bio/hc/en-us/articles/360026130851).
+The GVS beta workspaces in Terra are read-only, so you’ll need to clone the workspace to create a copy where you can upload your own data and run the workflow. Clone the workspace using the billing project you created in Step 2 (above) by following the instructions in the article [Make your own project workspace](https://support.terra.bio/hc/en-us/articles/360026130851).
```

Comment:
Can we be specific about using the exome workspace for exomes?  That's necessary right?